content/nldi_geoconnex.md

```diff
@@ -0,0 +1,114 @@
+
+For this new NLDI release, newly developed capabilities in the [nhdplusTools R package](https://code.usgs.gov/water/nhdplusTools) were used to find multiple nearby rivers for each site and avoid those with major differences in drainage area estimated at the gage and by the source hydrography dataset.
+
+It should be noted that the methods applied here were automated and issues should still be expected to exist. The these network locations are for data discovery and visualization. Any analytical or decision making use should be reviewed before assuming accuracy.
+
+geoconnex.us
+----------------------------
+
+https://geoconnex.us is a public domain registry system supported by the [Internet of Water](https://internetofwater.org/) team to facilitate use of linked open data about water on the Internet. A geoconnex.us identifier can be thought of like a [Digital Object Identifier (DOI)](https://www.doi.org/) in that it forwards (redirects) a web request to a different (less persistent) URL. For example,
+
+https://geoconnex.us/usgs/monitoring-location/08279500
+
+redirects to:
+
+https://waterdata.usgs.gov/monitoring-location/08279500/
+
+This provides a stable identifier that can be used in places where persistence is important, such as structured metadata and knowledge management. Because the redirect can change, it allows people implementing web services and systems to change domain and/or URL path structure while allowing persistent and robust indexing of the information in question. For more on the design of this arrangement, see the [first](https://docs.opengeospatial.org/per/18-097.html) and [second](https://docs.ogc.org/per/20-067.html) Environmental Linked Features Interoperability Experiment outcomes.
+
+Q: How does the NLDI and hydrographic addressing relate to geoconnes.ux?
+
+A: On the internet, a hydrographic address can be expressed as a link between a hydrographic feature and a located feature (such as a stream gage). Identifiers for both hydrographic and located features can be registered with geoconnex.us and indexed and discovered with the NLDI.
+
+Scope note
+-------------------------------
+
+**geoconnex.us is only a registry for identifiers.** It does not attempt to catalog or publish geospatial features or other data.  
+
+**The NLDI is only an index and set of data services.** It does not persist data other than the hydrographic network that makes up its index and allows it to provide data services.  
+
+These scope limitations are important to understand for two reasons:
+
+1) persistent and authoritative data systems such as waterdata.usgs.gov are required for the web architecture of geoconnex.us and the NLDI and
+2) the limited scope of the two systems means they can be used as building blocks that provide flexibility for systems that integrate with them.
+
+Reference gages
```

Comment:
as a general point, what is the scope of "reference gages"?

- Any surface water "monitoring location", including those with WQ measurements only?
- Any locations with >=1 stream stage or flow measurement, including one-off or irregular sampling activities?
- Any locations that might be characterized as "time series" generating stage and/or flow measurements

Reply:
Good question. I haven't made any firm decisions here. Not sure I want to get specific here.

I added:
> NOTE: For the time being "reference gages" are any location where information is collected along a river. As the project progresses, further definition of what kind of monitoring may be used to refine the definition of "gage".